lib/opFns.js

```diff
@@ -470,6 +470,8 @@ module.exports = {
     return loaded
   },
   DUP: function (runState) {
```

Comment:
Actually for `DUP` we could change the opcodes table to say takes 1 off and pushes 2 new items, but that makes it just more complex here.